src/layouts/Folders.jsx

```diff
@@ -58,6 +56,7 @@ const Folders = ({ match, location }) => {
     const [directoryFileSha, setDirectoryFileSha] = useState('')
     const [folderOrderArray, setFolderOrderArray] = useState([])
     const [parsedFolderContents, setParsedFolderContents] = useState([])
+    const [parsedFolderOutput, setParsedFolderOutput] = useState(true)
```

Comment:
Could we rename `parsedFolderOutput`? It's not immediately apparent that it's a boolean value for showing the folder